tfs/writer.py

```diff
@@ -64,56 +69,73 @@ def write_tfs(
         non_unique_behavior (str): behavior to adopt if non-unique indices or columns are found in the
             dataframe. Accepts `warn` and `raise` as values, case-insensitively, which dictates
             to respectively issue a warning or raise an error if non-unique elements are found.
-        validate (bool): Whether to validate the dataframe before writing it to file. Defaults to ``True``.
+        validate (str): Determines if and which validation will be performed before writing.
+            Defauts to `madx`, which will assert compatibility of the data with the ``MAD-X`` code.
+            If `madng` is given then compatibility with the ``MAD-NG`` code is checked, which has more
+            features. Accepted values are `madx`, `mad-x`, `madng` and `mad-ng`, case-insensitive. If
+            set to `None`, validation will be skipped. See the `tfs.frame.validate` function
+            for more information on the validation steps.
 
     Examples:
         Writing to file is simple, as most arguments have sane default values.
         The simplest usage goes as follows:
 
         .. code-block:: python
 
-            >>> tfs.write("filename.tfs", dataframe)
+            tfs.write("filename.tfs", dataframe)
 
         If one wants to, for instance, raise and error on non-unique indices or columns,
         one can do so as:
 
         .. code-block:: python
 
-            >>> tfs.write("filename.tfs", dataframe, non_unique_behavior="raise")
+            tfs.write("filename.tfs", dataframe, non_unique_behavior="raise")
```

Comment:
outdated, as one needs to activate validation first

Reply:
Changed to how to DO validation if the user wants to.